src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);
```

Comment:
Creating a new instance of 'Random' every time you need a random number is inefficient. Instead, create a single 'Random' instance as a class field and reuse it.